doc/neo-tree.txt

```diff
@@ -148,6 +148,38 @@ commands by adding this option before the Neotree config is run:
 >vim
   let g:neo_tree_remove_legacy_commands = 1
 <
```

Comment:
It's a little confusing to have this come after the deprecated commands and not immediately following the documentation for this particular function. The deprecated commands have actually been removed anyway and they should have been deleted from the docs. Might as well do that now.

```suggestion
```